[2025-04-08T23:13:19.106+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-04-08T23:13:19.127+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: extract_raw_data_pipeline.create_raw_table manual__2025-04-08T23:12:51.000773+00:00 [queued]>
[2025-04-08T23:13:19.138+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: extract_raw_data_pipeline.create_raw_table manual__2025-04-08T23:12:51.000773+00:00 [queued]>
[2025-04-08T23:13:19.138+0000] {taskinstance.py:2865} INFO - Starting attempt 1 of 1
[2025-04-08T23:13:19.273+0000] {taskinstance.py:2888} INFO - Executing <Task(SQLExecuteQueryOperator): create_raw_table> on 2025-04-08 23:12:51.000773+00:00
[2025-04-08T23:13:19.282+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=900) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-04-08T23:13:19.285+0000] {standard_task_runner.py:72} INFO - Started process 902 to run task
[2025-04-08T23:13:19.284+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'extract_raw_data_pipeline', 'create_raw_table', 'manual__2025-04-08T23:12:51.000773+00:00', '--job-id', '32', '--raw', '--subdir', 'DAGS_FOLDER/driven_data_pipeline.py', '--cfg-path', '/tmp/tmprqx7uxje']
[2025-04-08T23:13:19.286+0000] {standard_task_runner.py:105} INFO - Job 32: Subtask create_raw_table
[2025-04-08T23:13:19.507+0000] {task_command.py:467} INFO - Running <TaskInstance: extract_raw_data_pipeline.create_raw_table manual__2025-04-08T23:12:51.000773+00:00 [running]> on host 5a8aeda502d0
[2025-04-08T23:13:19.987+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='extract_raw_data_pipeline' AIRFLOW_CTX_TASK_ID='create_raw_table' AIRFLOW_CTX_EXECUTION_DATE='2025-04-08T23:12:51.000773+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-04-08T23:12:51.000773+00:00'
[2025-04-08T23:13:19.988+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-04-08T23:13:20.001+0000] {sql.py:266} INFO - Executing: 
        CREATE TABLE IF NOT EXISTS raw.raw_batch_data (
            person_name VARCHAR(100),
            user_name VARCHAR(100),
            email VARCHAR(100),
            personal_number NUMERIC, 
            birth_date VARCHAR(100), 
            address VARCHAR(100),
            phone VARCHAR(100), 
            mac_address VARCHAR(100),
            ip_address VARCHAR(100),
            iban VARCHAR(100),
            accessed_at TIMESTAMP,
            session_duration INT,
            download_speed INT,
            upload_speed INT,
            consumed_traffic INT,
            unique_id VARCHAR(100)
        );
    
[2025-04-08T23:13:20.009+0000] {base.py:84} INFO - Retrieving connection 'postgres_conn'
[2025-04-08T23:13:20.020+0000] {base.py:84} INFO - Retrieving connection 'postgres_conn'
[2025-04-08T23:13:20.025+0000] {sql.py:509} INFO - Running statement: 
        CREATE TABLE IF NOT EXISTS raw.raw_batch_data (
            person_name VARCHAR(100),
            user_name VARCHAR(100),
            email VARCHAR(100),
            personal_number NUMERIC, 
            birth_date VARCHAR(100), 
            address VARCHAR(100),
            phone VARCHAR(100), 
            mac_address VARCHAR(100),
            ip_address VARCHAR(100),
            iban VARCHAR(100),
            accessed_at TIMESTAMP,
            session_duration INT,
            download_speed INT,
            upload_speed INT,
            consumed_traffic INT,
            unique_id VARCHAR(100)
        );
    , parameters: None
[2025-04-08T23:13:21.462+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-04-08T23:13:21.472+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=extract_raw_data_pipeline, task_id=create_raw_table, run_id=manual__2025-04-08T23:12:51.000773+00:00, execution_date=20250408T231251, start_date=20250408T231319, end_date=20250408T231321
[2025-04-08T23:13:21.898+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-04-08T23:13:21.935+0000] {taskinstance.py:3900} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-04-08T23:13:22.250+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
